std/getopt.d

```diff
@@ -1773,3 +1773,37 @@ void defaultGetoptFormatter(Output)(Output output, string text, Option[] opt)
     getopt(args, "f|flag", "Boolean", &flag);
     assert(flag);
 }
+
+@system unittest  // Delegates as callbacks
```

Comment:
I'm not disagreeing that we need to add these unittest... but I think it's worth investigating where exactly it fails to be `@safe`. It might be a bug in the implementation,